.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -32,8 +32,6 @@ jobs:
         exec_model: ${{ fromJson(needs.set_wheel_build_matrix.outputs.exec_model) }}
         kokkos_version: ${{ fromJson(needs.set_wheel_build_matrix.outputs.kokkos_version) }}
         container_img: ["quay.io/pypa/manylinux2014_aarch64"]
-        exclude:
-          - exec_model: OPENMP
```

Comment:
Is this not required anymore?

Reply:
Not if we only build Kokkos-OPENMP for the wheels.